src/app/content/highlights/reducer.ts

```diff
@@ -145,8 +145,8 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
       };
     }
     case getType(actions.receiveHighlights): {
-      return {...state, highlights: [...state.highlights || [], ...action.payload],
-        summary: {...state.summary, loading: false},
+      return {...state, highlights: action.payload,
+        summary: {...state.summary},
```

Comment:
This `loading: false` is still needed:
![image](https://user-images.githubusercontent.com/31478212/74439041-bbf3b680-4e6b-11ea-889e-add8a32fbcd8.png)

Not sure why though, you could investigate or just leave it since this in no related to this pr.

Reply:
@PiotrKozlowski fixed in e871418, i know its not totally related to this issue, but lets leave it here anyway